api/hubble/accounts.mdx

```diff
@@ -0,0 +1,26 @@
+| last_modified_ledger | int64 | The ledger sequence number when the ledger entry was last modified. Deletions do not count as a modification and will report the prior modification sequence number<br /><br />- Natural Key<br />- Cluster Field<br />- Required Field<br /><br />#### Notes:<br />As an example, if an account updates a signer's weight at sequence 1234 and then decides to delete the signer at 2345, the deleted record will still have a modified sequence of 1234. The `last_modified_ledger` **is not** a proxy for entry deletion time and should not be used in such a manner. Deletion times can be approximated with `batch_run_date`. |
+| ledger_entry_change | int64 | Code that describes the ledger entry change type that was applied to the ledger entry.<br /><br />- Required Field<br /><br />#### Notes:<br />Not every ledger entry can be updated, some are only created or deleted. Pay attention to types that are not valid for certain ledger entries.<br /><br /><table> <thead> <tr> <th>Value</th> <th>Description</th> <th><strong>Not</strong> Valid For</th> </tr> </thead> <tbody> <tr> <td>0</td> <td>Ledger Entry Created</td> <td></td> </tr> <tr> <td>1</td> <td>Ledger Entry Updated</td> <td>claimable balances</td> </tr> <tr> <td>2</td> <td>Ledger Entry Deleted</td> <td></td> </tr> </tbody> </table> |
+| deleted | bool | Indicates whether the ledger entry (account, claimable balance, trust line, offer, liquidity pool) has been deleted or not. Once an entry is deleted, it cannot be recovered.<br /><br />All state tables maintain history for deleted ledger entries.<br /><br />- Required Field |
+| batch_id | string | String representation of the run id for a given DAG in Airflow. Takes the form of "scheduled__<batch_end_date>-<dag_alias>". Batch ids are unique to the batch and help with monitoring and rerun capabilities |
+| batch_run_date | datetime | The start date for the batch interval. When taken with the date in the batch_id, the date represents the interval of ledgers processed. The batch run date can be seen as a proxy of closed_at for a ledger. |
+| batch_insert_ts | timestamp | The timestamp in UTC when a batch of records was inserted into the database. This field can help identify if a batch executed in real time or as part of a backfill. The timestamp should not be used during ad hoc analysis and is useful for data engineering purposes. |
+| sponsor | string | The account address of the sponsor who is paying the reserves for this ledger entry.<br /><br />The following ledger entry types can be sponsored:<br /><br /> - accounts<br /> - account signers<br /> - claimable balances<br /> - trust lines<br /><br />#### Notes:<br />Sponsors of claimable balances are the accounts that created the balance. |
```

Comment:
Even though these fields are appended on to the end of the table, does it make sense to move up past batch fields?